packages/pluggableWidgets/accordion-native/src/components/AccordionGroup.tsx

```diff
@@ -32,21 +32,32 @@ export function AccordionGroup({
     visible,
     style
 }: AccordionGroupProps): ReactElement | null {
+    const isAccessible = group.accessible === "yes";
     return visible ? (
         <View style={style.container}>
             <Pressable
                 style={[style.header.container, icon === "left" && { flexDirection: "row-reverse" }]}
                 onPress={collapsible ? () => onPressGroupHeader(group, index) : null}
+                accessible={isAccessible}
```

Comment:
I am not sure if this is the right place to configure if the element is accessible or not, as if the accordion element is open by default then the children will be accessible still.